cron/internal/worker/main.go

```diff
@@ -176,7 +172,6 @@ func processRequest(ctx context.Context,
 	githubClient, gitlabClient clients.RepoClient, ossFuzzRepoClient clients.RepoClient,
 	ciiClient clients.CIIBestPracticesClient,
 	vulnsClient clients.VulnerabilitiesClient,
-	projectClient packageclient.ProjectPackageClient,
```

Comment:
Any reason this isn't being passed but the rest are?

Reply:
There isn't a WithProjectPackageClient option because there's only one client, so there's not currently a way of configuring another option. One will still get made in `pkg.Run`.

https://github.com/ossf/scorecard/blob/5f7cea363724ba03923eb07bdb4e6fa3b2927c15/pkg/scorecard.go#L388-L390

Reply:
Happy to make one for testing / mocking purposes if needed, but functionally isn't need here.

Reply:
Gotcha, I didn't realize it was the only Client that just had one possible implementation.